.azure/vs2022.yml

```diff
@@ -1558,6 +1558,7 @@ stages:
               restoreNugetPackages: false
               createLogFile: true
               msbuildArguments:
+                -restore
```

Comment:
Can you not use `restoreNugetPackages` and set it to `true` instead?

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/msbuild-v1?view=azure-pipelines

Reply:
Oh probably. The doc says that it's deprecated but I think it'll still be good enough for us because their workaround is more complicated:

> This option is deprecated. To restore NuGet packages, add a [NuGet](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/package/nuget) task before the build.